sematic/metrics/func_run_count.py

```diff
@@ -0,0 +1,31 @@
+# Standard Library
+from datetime import datetime
+from typing import Optional, Tuple
+
+# Third-party
+from sqlalchemy.orm import Query, Session, joinedload
+
+# Sematic
+from sematic.abstract_metric import AbstractMetric
+from sematic.db.models.run import Run
+from sematic.metrics.metric_point import MetricType
+
+
+class FuncRunCount(AbstractMetric):
+    """
+    System Metric to count the number of runs created.
+    """
+
+    @classmethod
+    def _get_name(self) -> str:
+        return "run_count"
+
+    @classmethod
+    def _get_metric_type(self) -> MetricType:
+        return MetricType.COUNT
+
+    def _get_value(self, run: Run) -> Optional[Tuple[datetime, float]]:
```

Comment:
I'm curious how this metric will work if we put a time limit on retention of metrics. Let's say we only store for 30 days--will this then need to be changed in the UI to be "Runs in the last 30 days"?

Reply:
We agreed during Design Review to only apply the retention policy to User Metrics.